sdcm/nemesis.py

```diff
@@ -513,6 +513,25 @@ def disrupt_switch_between_PasswordAuthenticator_and_SaslauthdAuthenticator_and_
         else:
             raise UnsupportedNemesis("SaslauthdAuthenticator can't work without saslauthd environment")
 
+    def disrupt_random_set_compression_on_all_dbnodes(self):
```

Comment:
change to:
"disrupt_rolling_config_change_internode_compression"

Reply:
done